File: matiasmasso/Mat.NET/DUI/DUI/GuidNom.vb
Public Class Guidnom
    Property Guid As Guid
    Property Nom As String
End Class
Public Class Guidnom2
    'per iMat, JSONDecoder de XCode ha de rebre les propietats en minuscules
    Property guid As Guid
    Property nom As String
End Class

Public Class Guidnom3
    'per iMat antic, JSONDecoder de XCode ha de rebre les propietats en majuscules
    Property Guid As Guid
    Property Nom As String
End Class

File: matiasmasso/Mat.NET/DTO/Models/Base/GuidNom.cs
using System;

namespace DTO.Models.Base
{
    public class GuidNom
    {
        public Guid Guid { get; set; }
        public string Nom { get; set; }

        public GuidNom() { }
        public GuidNom(Guid guid, string nom = "")
        {
            Guid = guid;
            Nom = nom;
        }
    }
}
